mobile/src/androidTest/java/ch/epfl/sdp/mobile/ui/features/social/StatefulFollowingScreenTest.kt

```diff
@@ -0,0 +1,42 @@
+package ch.epfl.sdp.mobile.ui.features.social
+
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createComposeRule
+import ch.epfl.sdp.mobile.data.api.AuthenticationApi
+import ch.epfl.sdp.mobile.data.api.ProfileColor
+import ch.epfl.sdp.mobile.ui.features.home.StatefulHome
+import ch.epfl.sdp.mobile.ui.i18n.setContentWithLocalizedStrings
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.emptyFlow
+import kotlinx.coroutines.flow.flow
+import org.junit.Rule
+import org.junit.Test
+
+
+class StatefulFollowingScreenTest {
+    @get:Rule
+    val rule = createComposeRule()
+
+
+    @Test
+    fun defaultMode_displayCorrectFollowers() {
+        val mockUser = mockk<AuthenticationApi.User.Authenticated>()
+        every { mockUser.following } returns flow {
+            emit(listOf<AuthenticationApi.Profile>(
+                object : AuthenticationApi.Profile {
+                    override val emoji: String
+                        get() = ":>"
+                    override val name: String
+                        get() = "Hans Peter"
+                    override val backgroundColor: ProfileColor
+                        get() = ProfileColor.Pink
+                }
+            )
+            )
+        }
```

Comment:
Tip : you can use `flowOf(listOf(...))` instead of `flow { emit (...) }` if you know all the values your flow will emit.

Reply:
Merci 👍